calculator_test.go

```diff
@@ -1,24 +1,173 @@
-package calculator_test
+package calculator
 
 import (
-	"calculator"
+	"math"
 	"testing"
 )
 
 func TestAdd(t *testing.T) {
 	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
-	if want != got {
-		t.Errorf("want %f, got %f", want, got)
+	var testCases = []struct {
+		a    float64
+		b    []float64
+		want float64
+		name string
+	}{
+		{20, []float64{1}, 21, "Adding two positive numbers yields a positive number"},
```

Comment:
We might put the name first, mightn't we—then it's a little easier to read in logical sequence.

Reply:
We sure might! Love it. Combined with moving the name of the case to be first, it is much readable now.